airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/GenerateInputActivityImpl.java

```diff
@@ -109,7 +116,8 @@ public GeneratedJobInput getSyncWorkflowInput(final SyncInput input) {
           .withAttemptId((long) attempt)
           .withDockerImage(config.getSourceDockerImage())
           .withProtocolVersion(config.getSourceProtocolVersion())
-          .withIsCustomConnector(config.getIsSourceCustomConnector());
+          .withIsCustomConnector(config.getIsSourceCustomConnector())
+          .withAllowedHosts(configReplacer.getAllowedHosts(sourceDefinition.getAllowedHosts(), config.getSourceConfiguration()));
```

Comment:
Note to self: this affects the intended plan for part 2 of the configuration update project. (it was proposed to build IntegrationLauncherConfigs separately at the beginning to re-use for the check and sync, but because this now depends on the source/destination configuration we should keep building the IntegrationLauncherConfigs whenever we generate the check and sync inputs)